learning-development/python.qmd

```diff
@@ -2,32 +2,68 @@
 title: "Python"
 ---
 
-<p class="text-muted">Guidance and tips for using Python</p>
+<p class="text-muted">
 
----
+Guidance and tips for using Python
+
+</p>
```

Comment:
Found this on another branch yesterday, the muted attribute seems to fail if it's not the same line, can you smush the tags and text for this subtitle all onto line 5? That should mute the text colour?

Here's what it looks like locally:

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/ed1f337d-cc40-46da-8028-99af11a8a75c)


Versus the Git page:

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/a3539cad-0471-4db1-a3ca-ba289372abf2)